config.yaml

```diff
@@ -0,0 +1,6 @@
+benchmarks:
+    - freetype2-2017
+    - bloaty_fuzz_target
```

Comment:
bloaty_fuzz_target is a oss-fuzz benchmark.
The next group of commits will resolve this issue.
If we change this into libpng-1.2.56. Then it should work now.

Reply:
yes, let's change it

Reply:
Done.